gcloud-java-compute/src/test/java/com/google/gcloud/compute/it/ITComputeTest.java

```diff
@@ -447,4 +446,190 @@ public void testListZonesWithFilter() {
     assertEquals(ZONE, zoneIterator.next().zoneId().zone());
     assertFalse(zoneIterator.hasNext());
   }
+
+  @Test
+  public void testListGlobalOperations() {
```

Comment:
Getting/deleting operation tests will arrive as soon as we have other Compute resources whose creation/deletion starts an operation (we will have to poll that operation to know when the resource is available). `gcloud-devel` already has plenty of operations.